src/index.ts

```diff
@@ -153,6 +149,34 @@ export default class Browserbase {
     return await response.json()
   }
 
+  async completeSession(sessionId: string): Promise<Session> {
+    if (!sessionId || sessionId === '') {
```

Comment:
This is very unlikely to happen with TypeScript, and we don't perform similar checks on other methods. 
we check the projectID below for other reasons.

Reply:
Not everyone uses TypeScript 😇 so we should do some runtime validation to prevent API calls that will never succeed

Reply:
i'm not sure we can rely on the types here; a user can pass in whatever they want to this method. we probably should perform similar checks on others or at least know how we're failing when we get input we don't expect. maybe time for https://fast-check.dev/?

Reply:
zod would be the most straightforward or leveraging TypeScript assert functions for a lightweight approach : https://github.com/microsoft/TypeScript/pull/32695